server/index.js

```diff
@@ -1,5 +1,5 @@
 const express = require('express');
-let port = 3000;
+let port = 1028;
 let app = express();
 const parser = require('body-parser');
```

Comment:
You should keep all of your require statements up at the top of your file. I'd recommend installing a linter ASAP to avoid styling issues.